components/common/circular_chart/circular_chart.scss

```diff
@@ -0,0 +1,47 @@
+.CircularChart {
+    .circular-chart {
+        display: block;
+        margin: 5px auto;
+
+        &.warning .circle {
+            stroke: var(--away-indicator);
+        }
+
+        &.success .circle {
+            stroke: var(--denim-status-online);
+        }
+
+        &.info .circle {
+            stroke: var(--sidebar-text-active-border);
+        }
+
+        &.error .circle {
+            stroke: var(--error-text);
+        }
```

Comment:
to be consistent we should use the `--sys-*` variables here as well

Reply:
the idea is that this component is common, so for general purpose these will be the available colors, then in the console, we style it using the `--sys-`ones as it is defined in `dashboard.scss lines 73 to 85`